source/includes/administration/_environments.md

```diff
@@ -80,7 +80,7 @@ Attributes | &nbsp;
 ```shell
 # Retrieve visible environment
 
-curl "https://cloudmc_endpoint/v1/environment/487a2745-bb8a-44bc-adb1-e3b048f6def2" \
+curl "https://cloudmc_endpoint/v2/environments/487a2745-bb8a-44bc-adb1-e3b048f6def2" \
```

Comment:
this change seems odd are you sure its environments not environment?

Reply:
yes, I've tested it